lib/dor/text_extraction/abbyy/file_watcher.rb

```diff
@@ -22,7 +22,7 @@ class FileWatcher
         # rubocop:disable Metrics/AbcSize
         def initialize(
           logger: Logger.new($stdout),
-          workflow_updater: Dor::TextExtraction::WorkflowUpdater,
+          workflow_updater_class: Dor::TextExtraction::WorkflowUpdater,
           listener_options: {}
```

Comment:
Any reason for this change from `workflow_updater` to `workflow_updater_class` here and in the other references in this file?  It doesn't seem to match what was added in `lib/dor/text_extraction/speech_to_text_create_done_handler.rb` below.

Reply:
ah, yeah, this was me feeling pedantic and scratching that itch.

the difference between this and `speech_to_text_create_done_handler.rb` is that this class constructor takes the class to instantiate when initializing `@workflow_updater`.  but i still left the actual instance var name as `workflow_updater` like before.  it's just the param name that changes to be more accurate (since it's a class to be instantiated, and it felt nicer to me to not refer to the class on which you call `.new` by the same name as the instance variable that comes from that).

iirc nothing actually overrides the default param value, so maybe i should've just removed the param entirely 🤷 (i might do that in a follow-up PR, just to reduce unnecessary complexity, easy enough to turn back to parameterize this if we need to later).

because nothing that uses it actually sends in an alternate to `WorkflowUpdater`, i didn't see the need to parameterize that in `speech_to_text_create_done_handler.rb`, so it just always does `@workflow_updater = Dor::TextExtraction::WorkflowUpdater.new(logger:)`.